[ADD] stock_picking_batch_validate_confirm (#1589)
OCA/stock-logistics-workflow: stock_picking_batch_validate_confirm/readme/USAGE.md

To use this module, you need to:

1. Go to Inventory App > Configuration > Warehouses.
1. Select warehouse or create new.
1. Select *Send goods in output and then deliver (2 steps)*.
1. Create some sale orders with storable product.
1. Validate some picks generated by sale orders but not all.
1. Go to Inventory App > Operations > Batch Transfer.
1. Create new Batch Transfer and add deliveries (out) generated by sale orders.
1. Set quantities done.
1. *Validate* batch.

> 1. A Wizard show pending origin moves, i.e. moves from picks not done yet.

Reviewer: ```suggestion
1. A Wizard shows pending origin moves, i.e. moves from picks not done yet.
```

> 1. Click on *Validate* in wizard if you still want validate the batch.

Reviewer: ```suggestion
1. Click on *Validate* in wizard if you still want to validate the batch.
```

> 1. Click on *Cancel* in wizard if you don't want validate the batch untill all origin moves is done.

Reviewer: ```suggestion
1. Click on *Cancel* in wizard if you don't want to validate the batch until all origin moves are done.
```
